index.bs

```diff
@@ -2776,6 +2786,26 @@ The following abstract operations support the implementation and manipulation of
     |view|, |readIntoRequest|).
 </div>
 
+<div algorithm>
+ <dfn abstract-op lt="ReadableStreamBYOBReaderRelease"
+ id="readable-stream-byob-reader-release">ReadableStreamBYOBReaderRelease(|reader|)</dfn>
```

Comment:
No hand-crafted id=""s for new abstract-ops

Reply:
Dangit, must have copy-pasted this from somewhere. 😛